### PR TITLE
Update the card details class based on v68

### PR DESCRIPTION
**Pull request by AlexandrosMor**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

**Review by zaiddreakh — DISMISSED** [review]

File: Adyen/Model/Checkout/CardDetails.cs [hunk 1]
```diff
@@ -19,7 +18,6 @@
 //  Copyright (c) 2020 Adyen B.V.
```
Comment (wboereboom) [inline Adyen/Model/Checkout/CardDetails.cs:18]:
```suggestion
//  Copyright (c) 2022 Adyen N.V.
```

**Review by wboereboom — CHANGES_REQUESTED** [review]
Only file header update is missing. All seems in order apart from that 👍

**Comment by AlexandrosMor** [comment]
We are pla

File: Adyen/Model/Checkout/CardDetails.cs [hunk 2]
```diff
@@ -33,367 +31,551 @@
 
 namespace Adyen.Model.Checkout
 {
-    /// <summary>
+     /// <summary>
```
Comment (wboereboom) [inline Adyen/Model/Checkout/CardDetails.cs:34]:
```suggestion
    /// <summary>
```

**Review by zaiddreakh — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Review by zaiddreakh — APPROVED** [review]

**Pull request merged**